src/main/java/io/apimatic/core/types/http/response/Dynamic.java

```diff
@@ -136,9 +133,8 @@ public double parseAsDouble() throws ParseException {
 
     /**
      * Parse response as string.
-     * 
-     * @return Parsed value
      * @throws ParseException Signals if a parse exception occured
+     * @return Parsed value
```

Comment:
Nits, Non-blocker: `@throws` should be the last line of the method summary.

Reply:
`@throws` Moved at the end of the method summary